src/frontend/src/binder/select.rs

```diff
@@ -238,6 +244,55 @@ impl Binder {
         Ok((select_list, aliases))
     }
 
+    /// `bind_get_user_by_id_select` binds a select statement that returns a single user name by id,
+    /// this is used for function `pg_catalog.get_user_by_id()`.
+    pub fn bind_get_user_by_id_select(&mut self, input: &ExprImpl) -> Result<BoundSelect> {
+        let select_items = vec![ExprImpl::InputRef(Box::new(InputRef::new(
```

Comment:
FYI `ExprImpl::InputRef(Box::new(e))` can be replaced with `e.into()` for all expr variants.